ui/app/components/project-input-variables/list.ts

```diff
@@ -14,7 +14,7 @@ interface ProjectSettingsArgs {
 export default class ProjectInputVariablesListComponent extends Component<ProjectSettingsArgs> {
   @service api!: ApiService;
   @service router!: RouterService;
-  @service flashMessages!: PdsFlashMessages;
+  @service flashMessages!: FlashMessagesService;
```

Comment:
This is super subtle, but strictly speaking we also need to update these `@service` decorators to tell Ember exactly which service we want to inject.

```suggestion
  @service('pds-flash-messages') flashMessages!: FlashMessagesService;
```

The service decorator doesn't normally need this extra bit of info because it can infer the name of the service from the name of the field it’s decorating (i.e. `flashMessages` → `flash-messages`). Because the name of the service now has `pds-` at the front, `@service` on its own would infer the wrong thing.